### ISSUE TITLE
[SUGGESTION] Update guild setup and database

### ISSUE THREAD

**Issue by PhantomNimbi** [galnir/Master-Bot#554]
**Explain your suggestion**

Yo, so I got an idea that might be worth looking into later if you guys are up for it. My idea is basically as the title suggests. To consolidate all of the guild settings commands into a single set of commands/subcommands.

Basically it would work more or less like so.

Twitch Announcer:
```js
[prefix]set twitch.alert [enable/disable]
[prefix]set twitch.alert [twitch_channel] [discord_channel_id/discord_channel_name] [time] [message]
```

Welcomer:
```js
[prefix]set welcomer [enable/disable]
[prefix]set welcomer [discord_channel_name/discord_channel_id] [title] [message] [width] [height] [image_url]
```

Guild Prefix:
```js
[prefix]set guild.prefix [prefix]
```

Naturally this would also use a slightly updated SQLiite database. Instead of Quick DB it would use MySQL to store guild and channel ids and so on. The SQLite Databases would be saved into a data folder located in the root of the bot directory.

Lastly I would like to suggest giving support for multiple Twitch channels instead of just one Twitch channel.

**Comment by Bacon-Fixation** [comment]
I can play around with a `set` command to control the twitch announcer and welcome message.

You can currently enable and disable modules, and change the prefix per server with `commandState: true` and `prefix: true` in the index.js, look into #384 for more info <3

not sure about the change in DB tho

Much Love
-Bacon

**Comment by Bacon-Fixation** [comment]
I wasn't able to reliably consolidated all the Settings commands with a `!set`
welcome-settings and twitch-announcer-settings needed different types and number of arguments
and wasn't able to make it work at all if I deleted one of them from the folder (say if someone deleted the twitch announcer.js and did a `!set twitch-announcer`), and put it on hold for now

but I have revived the work on #384 and reworking it to save the changes to the DB and reload them at start up
adding commands for per server changes like Prefix, Enable, and Disable commands 
https://github.com/Bacon-Fixation/Master-Bot/tree/Settings-Commands-Prototype 
basically guild only versions of discordjs-commando's commandState commands that save to the DB

I'm not sure if I should make this a PR, if I hit another road block I will abandon it again 
Just wanted to let you know that I am trying to fulfill this request if I can.
This may take a long time to complete, while I figure out what I can.

also adding the ability to have more than one streamer is possible it did get a proof of concept working, but I am holding off till I can get the Twitch Announcer's Timers to return after a reboot. 


Kinda what my Road Map looks like for the near future

Much Love
-Bacon
